### PR TITLE
Fixed two memory leaks

### PR DESCRIPTION
**Pull request by tsujan**

### CODE REVIEW

File: src/qtxdg/xdgmimeappsglibbackend.cpp [hunk 1]
```diff
@@ -42,6 +42,9 @@ static QList<XdgDesktopFile *> GAppInfoGListToXdgDesktopQList(GList *list)
                 if (df->load(file) && df->isValid()) {
                     dl.append(df);
                 }
+                else {
```
Comment (luis-pereira) [inline src/qtxdg/xdgmimeappsglibbackend.cpp:45]: Coding style not followed.

Comment (tsujan) [reply src/qtxdg/xdgmimeappsglibbackend.cpp:45]: > Coding style not followed.

`} else {` or something else? I did my best to follow the style ;)

Comment (luis-pereira) [reply src/qtxdg/xdgmimeappsglibbackend.cpp:45]: ```} else {```

there isn't any "example" in the all file.
Then merge it.

Comment (tsujan) [reply src/qtxdg/xdgmimeappsglibbackend.cpp:45]: Will change and merge it. Thanks!

**Pull request merged**